### PR TITLE
Feature GAL-647

### PR DESCRIPTION
**Pull request by avi312NHS**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by zainm97 — DISMISSED** [review]

File: terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js [hunk 1]
```diff
@@ -125,7 +131,8 @@ export async function updateRecord(record, batchId, client) {
     TableName: `${ENVIRONMENT}-Population`,
     UpdateExpression: `SET
       #IDENTIFIED_TO_BE_UPDATED = :to_be_invited,
-      #BATCH_ID = :batch`,
+      #BATCH_ID = :batch,
+      #CREATED_BY: created`,
   };
```
Comment (Amar-NHS) [inline terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js:136]: think you missed the "=" above. --> `#CREATED_BY = :created`

**Review by Amar-NHS — COMMENTED** [review]
got 1 comment to review

**Review by Aymene-n — COMMENTED** [review]
Once you resolve @Amar-NHS comment I can approve.

**Review by Abdul-NHS — APPROVED** [review]

File: terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js [hunk 2]
```diff
@@ -46,15 +48,17 @@ export const handler = async (event, context) => {
...
-        (curr, acc) => {if (curr.value === SUCCESSFULL_REPSONSE) return acc + 1}
-        ,0
-      )
-      console.log(`Only ${successfulRecords}/${responsePopulation.length} persons were successfully updated. Contact third line support to investigate`);
+    } else {
+      const successfulRecords = responsePopulation.reduce((curr, acc) => {
+        if (curr.value === SUCCESSFULL_REPSONSE) return acc + 1;
+      }, 0);
+      console.log(
+        `Only ${successfulRecords}/${responsePopulation.length} persons were successfully updated. Contact third line support to investigate`
```
Comment (andrew-cleveland) [inline terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js:60]: Can you add in Error: to the begining of this string so we can catch it more easily with filters

**Review by andrew-cleveland — DISMISSED** [review]

**Review by Aymene-n — DISMISSED** [review]

**Review by chenglawson1 — APPROVED** [review]

**Review by Amar-NHS — APPROVED** [review]

**Pull request merged**